src/pybind/mgr/dashboard_v2/controllers/task.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from ..tools import ApiController, AuthRequired, RESTController, TaskManager
+
+
+@ApiController('task')
+@AuthRequired()
+class TaskController(RESTController):
+    def list(self, namespace=None):
```

Comment:
Namespaces are useful to group related tasks. For instance, to group all tasks related to a single component like RBD `rbd/*`.
Since the frontend is the main entity to consume the list of tasks, working with a "task name" is difficult. The frontend memory is volatile and might easily lose the task name upon refresh, therefore it is easier to use namespaces.

Also, you can define a unique namespace to mimic the same behavior of a "task name" or "task id".

The `namespace` parameter in this function accepts a glob expression, which makes it pretty good for filtering groups of tasks.

Reply:
Just renaming `namespace` to `name` or `task_name` would make things clearer. Especially when moving the name to the task definition. We're talking about the same thing here.

Reply:
I don't have strong feelings either way, but `namespace` does sound a bit better than `name` or `task_name` IF `namespace` here is supposed to allow more than just a specific task unique identifier; `name` and `task_name`, on the other hand, seem to convey something a bit too specific.

However, I don't particularly like `namespace` for this particular case, but I don't have a better name for it, so...